pkg/reconciler/contour/config/contour.go

```diff
@@ -70,6 +75,40 @@ func NewContourFromConfigMap(configMap *corev1.ConfigMap) (*Contour, error) {
 		return nil, err
 	}
 
+	cors, ok := configMap.Data[corsPolicy]
+	if ok {
+		if err := yaml.Unmarshal([]byte(cors), &contourCORSPolicy); err != nil {
+			return nil, err
```

Comment:
This needs a better message because you won't know the `cors-policy` field wasn't valid

Reply:
`error unmarshaling JSON: while decoding JSON: json: cannot unmarshal number into Go struct field CORSPolicy.allowCredentials of type bool` What more info you looking for?

Reply:
Oh cool the yaml library returns a good error message